build.gradle

```diff
@@ -57,7 +56,7 @@ repositories {
 }
 
 final requiredJavaVersion = "8"
-final htsjdkVersion = System.getProperty('htsjdk.version','2.18.0')
+final htsjdkVersion = System.getProperty('htsjdk.version','2.18.1')
```

Comment:
Can you add an integration test with one of the CRAMs in question that fails with 2.18.0 and passes with 2.18.1?

Reply:
Joel tried to build one and failed....

Reply:
I think he might have figured it out -- @jmthibault79 can you comment?

Reply:
Nope.  https://github.com/samtools/htsjdk/issues/1236

Reply:
I think it would be hard to generate one of these with htsjdk even if we had a fix for samtools/htsjdk#1236, since htsjdk doesn't use that (BetaEncoding) for alignment start; it uses ExternalIntegerEncoding. Its probably samtools uses (or once used) that encoding for alignment start.

Reply:
Alright, fair enough -- we can merge this as-is then.